### PR TITLE
Create sqoop.txt

### PR DESCRIPTION
**Pull request by saimanoj1234**

### CODE REVIEW

File: sqoop.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+sqoop tool rdms to hdfs import
```
Comment (kancharlapavan) [inline sqoop.txt:1]: wrong format

**Review by kancharlapavan — CHANGES_REQUESTED** [review]

**Review by kancharlapavan — APPROVED** [review]

**Review by kancharlapavan — APPROVED** [review]

**Review by kancharlapavan — APPROVED** [review]
okk

File: sqoop.txt (hunk 1 above)
Comment (kancharlapavan) [inline sqoop.txt:1]: changed required

**Review by kancharlapavan — CHANGES_REQUESTED** [review]

**Comment by saimanoj1234** [comment]
changes done

File: sqoop.txt (hunk 1 above)
Comment (saimanoj1234) [reply sqoop.txt:1]: okay ill change

**Pull request merged**